
# This is a patch for Devel-REPL-1.003011.orig to update it to Devel-REPL-1.003011
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "Devel-REPL-1.003011.orig\Makefile.PL" "Devel-REPL-1.003011\Makefile.PL"
Index: ./Makefile.PL
*** ./Makefile.PL	Sun May 23 00:29:09 2010
--- ./Makefile.PL	Thu Jul 15 23:00:20 2010
***************
*** 41,47 ****
      'File::Next' => 0;
  
  feature 'Interrupt plugin - traps SIGINT to kill long-running lines',
!     -default => 1,
      'Sys::SigAction' => 0;
  
  feature 'Keywords completion driver - tab complete Perl keywords and operators',
--- 41,47 ----
      'File::Next' => 0;
  
  feature 'Interrupt plugin - traps SIGINT to kill long-running lines',
!     -default => 0,
      'Sys::SigAction' => 0;
  
  feature 'Keywords completion driver - tab complete Perl keywords and operators',
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Thu Jul 15 23:00:59 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "Makefile.PL" 1912 1279256420 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Thu Jul 15 23:00:59 2010] ####
#### Patch checksum: 55 1972 56196 ####
#### Checksum: 73 2676 48459 ####
